nvdaHelper/vbufBackends/mshtml/mshtml.cpp

```diff
@@ -23,35 +23,26 @@ This license can be found at:
 #include <sstream>
 #include <vbufBase/backend.h>
 #include <vbufBase/utils.h>
+#include <remote/dllmain.h>
 #include <common/log.h>
 #include "node.h"
 #include "mshtml.h"
 
 using namespace std;
 
-HINSTANCE backendLibHandle=NULL;
-UINT WM_HTML_GETOBJECT;
-
-BOOL WINAPI DllMain(HINSTANCE hModule,DWORD reason,LPVOID lpReserved) {
-	if(reason==DLL_PROCESS_ATTACH) {
-		_CrtSetReportHookW2(_CRT_RPTHOOK_INSTALL,(_CRT_REPORT_HOOKW)NVDALogCrtReportHook);
-		backendLibHandle=hModule;
-		WM_HTML_GETOBJECT=RegisterWindowMessage(L"WM_HTML_GETOBJECT");
-	}
-	return TRUE;
-}
+UINT WM_HTML_GETOBJECT=RegisterWindowMessage(L"WM_HTML_GETOBJECT");

```

Comment:
This potentially subtly changes the order/timing of when this happens. This might not be a problem, but it's worth being aware of. If possible, I would prefer to see an explicit initialisation function.